mappings/net/minecraft/entity/player/PlayerInventory.mapping

```diff
@@ -44,7 +44,10 @@ CLASS net/minecraft/class_136 net/minecraft/entity/player/PlayerInventory
 	METHOD method_690 getEmptySlot ()I
 	METHOD method_691 setHeldItem (IZ)V
 		COMMENT Will replace currently held item, removing it
+		COMMENT
+		COMMENT unused parameter is true if interaction manager is instanceof TestInteractionManager
 		ARG 1 itemId
+		ARG 2 unused
```

Comment:
If this is for when interaction manager is instance of TestInteractionManager can't the variable be called `test` because in theory a mixin would use this and i feel its weird to just use `unused` as the variable.